MainWindow.cpp:149:9: error: no matching function for call to 'FileBrowser::FileBrowser...

@Lukas-W, could this be caused by the enhancements in #1058 / #1131?  No problems with `gcc`, just `mingw`/windows building.

```
[ 26%] Building CXX object CMakeFiles/lmms.dir/src/gui/MainWindow.obj
/home/tres/lmms/src/gui/MainWindow.cpp: In constructor 'MainWindow::MainWindow()':
/home/tres/lmms/src/gui/MainWindow.cpp:149:9: error: no matching function for call to 'FileBrowser::FileBrowser(QString, const char [2], QString, QString, QPixmap, QSplitter*&, bool)'
         ) );
         ^
/home/tres/lmms/src/gui/MainWindow.cpp:149:9: note: candidates are:
In file included from /home/tres/lmms/src/gui/MainWindow.cpp:51:0:
/home/tres/lmms/include/FileBrowser.h:51:2: note: FileBrowser::FileBrowser(const QString&, const QString&, const QString&, const QPixmap&, QWidget*, bool)
  FileBrowser( const QString & directories, const QString & filter,
  ^
/home/tres/lmms/include/FileBrowser.h:51:2: note:   candidate expects 6 arguments, 7 provided
/home/tres/lmms/include/FileBrowser.h:47:7: note: FileBrowser::FileBrowser(const FileBrowser&)
 class FileBrowser : public SideBarWidget
       ^
/home/tres/lmms/include/FileBrowser.h:47:7: note:   candidate expects 1 argument, 7 provided
make[2]: *** [CMakeFiles/lmms.dir/src/gui/MainWindow.obj] Error 1
make[1]: *** [CMakeFiles/lmms.dir/all] Error 2
make: *** [all] Error 2
```

--- a/src/gui/MainWindow.cpp
+++ b/src/gui/MainWindow.cpp
@@ -133,12 +133,12 @@ MainWindow::MainWindow() :
 	sideBar->appendTab( new FileBrowser( root_paths.join( "*" ), "*",
 #ifdef LMMS_BUILD_WIN32
 							tr( "My computer" ),
-#endif
-#ifdef LMMS_BUILD_APPLE
+#elif LMMS_BUILD_APPLE
 							tr( "Volumes" ),
 #else
 							tr( "Root directory" ),
 #endif
+
 					embed::getIconPixmap( "computer" ).transformed( QTransform().rotate( 90 ) ),
 							splitter,
 #ifdef LMMS_BUILD_WIN32

